apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -138,6 +139,7 @@ export default function TemplateEditorPage() {
 
   return (
     <>
+      <TourProvider />
```

Comment:
the tour provider doesn't expose its context to editor children, but only to hint components that are rendered inside with portals and have access to it